Tribler/Test/Core/Modules/RestApi/test_display_endpoint.py

```diff
@@ -7,8 +16,19 @@ class TestDisplayEndpoint(AbstractApiTest):
     Test the DisplayEndpoint, the endpoint from which you can retrieve aggregated data from the multichain.
     """
 
-    def setUpPreSession(self):
-        super(TestDisplayEndpoint, self).setUpPreSession()
+    @blocking_call_on_reactor_thread
+    @inlineCallbacks
+    def setUp(self, autoload_discovery=True):
```

Comment:
@devos50 What do you think of this test? Is this the proper way to approach this?